modules/show.js

```diff
@@ -0,0 +1,18 @@
+// const date = document.getElementById('date');
+const listBtn = document.getElementById('list-link');
+const listSection = document.querySelector('.list-section');
+const addBtn = document.querySelector('#add-link');
+const addSection = document.getElementById('add-book');
+const contactBtn = document.querySelector('#contact-link');
+const contactSection = document.querySelector('.contact-section');
+
+// function showDateTime() {
+//   const dateNew = new Date();
+//   date.innerHTML = dateNew.toLocaleString('en-GB');
+// }
+// showDateTime();
+// setInterval(showDateTime, 1);
```

Comment:
- [ ] Kindly remove commented unused pieces of code from the project for a cleaner, more professional code 💯